clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -140,51 +159,59 @@ func (s *schedulerLatencyMeasurement) getSchedulingLatency(c clientset.Interface
 		return nil, err
 	}
 
-	e2eSchedulingMetricHist := measurementutil.NewHistogram(nil)
+	e2eSchedulingDurationHist := measurementutil.NewHistogram(nil)
 	schedulingAlgorithmDurationHist := measurementutil.NewHistogram(nil)
-	for _, sample := range samples {
-		if sample.Metric[model.MetricNameLabel] == e2eSchedulingDurationMetricName {
-			measurementutil.ConvertSampleToHistogram(sample, e2eSchedulingMetricHist)
-			continue
-		}
-		if sample.Metric[model.MetricNameLabel] == schedulingAlgorithmDurationMetricName {
```

Comment:
we still need this one, that is why ```schedulingLatency``` in https://github.com/kubernetes/perf-tests/pull/1081#issuecomment-607825761 are zeros.